package.json

```diff
@@ -78,16 +78,16 @@
     "react-table": "^7.7.0",
     "react-test-renderer": "^16.9.0",
     "react-virtualized": "^9.21.1",
+    "react-virtualized-auto-sizer": "^1.0.5",
```

Comment:
This change doesn't seems to relate to the scope of this pr, is it really necessary ?

Reply:
there is no change, npm decided to change the position on his own when i did the `npm i`